From: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=504660#25 
Description: declare pointer with a different type to reflect a change
 in gtkmm API. Fixes FTBFS.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=577305
---
 src/mainwnd.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- seq24-0.9.0.orig/src/mainwnd.cpp
+++ seq24-0.9.0/src/mainwnd.cpp
@@ -528,7 +528,7 @@ mainwnd::file_import_dialog( void )
 
     dialog.set_current_folder(last_used_dir);
 
-    HButtonBox *btnbox = dialog.get_action_area(); 
+    ButtonBox *btnbox = dialog.get_action_area();
     HBox hbox( false, 2 );
 
     m_adjust_load_offset = manage( new Adjustment( 0, -(c_max_sets - 1),
